exporting/aws_kinesis/README.md

```diff
@@ -0,0 +1,49 @@
+# Using Netdata with AWS Kinesis Data Streams
+
+## Prerequisites
+
+To use AWS Kinesis for metric collecting and processing, AWS SDK for C++ should be
```

Comment:
maybe change the last part of the sentence to, "...you must [install](https://docs.aws.amazon.com/en_us/sdk-for-cpp/v1/developer-guide/setup.html) AWS SDK for C++ first.

Reply:
Done